src/cargo/util/command_prelude.rs

```diff
@@ -152,6 +152,12 @@ pub trait AppExt: Sized {
         self._arg(opt("manifest-path", "Path to Cargo.toml").value_name("PATH"))
     }
 
+    fn arg_crate_type(self) -> Self {
+        self._arg(
+            opt("crate-type", "Override crate type for all libraries").value_name("CRATE-TYPE"),
```

Comment:
```suggestion
            opt("libs-crate-type", "Override crate type for all libraries").value_name("CRATE-TYPE"),
```

Reply:
Is this necessary? Since crate-type [always applies to libraries and examples](https://doc.rust-lang.org/cargo/reference/cargo-targets.html#the-crate-type-field). I think there are two options:

1. Call it `--crate-type` and apply the modification to both the library _and_ all example targets. (This PR currently only applies it to libraries, not examples).
2. Call it `--libs-crate-type` and apply the modification only to the library target.